tests/mocked_plans/sample_plans/9eb9af6a-b919-4cf9-8f2f-9df16a1556be.yaml

```diff
@@ -163,6 +159,9 @@ plan: |
             Upgrade plan for 'octavia-ovn-chassis' to 'victoria'
                 WARNING: Changing 'octavia-ovn-chassis' channel from latest/stable to 22.03/stable. This may be a charm downgrade, which is generally not supported.
                 Wait for up to 300s for app 'octavia-ovn-chassis' to reach the idle state
+            Upgrade plan for 'ovn-chassis' to 'victoria'
+                WARNING: Changing 'ovn-chassis' channel from latest/stable to 22.03/stable. This may be a charm downgrade, which is generally not supported.
+                Wait for up to 300s for app 'ovn-chassis' to reach the idle state
```

Comment:
**nitpick:** 

The same issue, the `ovn-chassis` shouldn't belong to Control plane subordinates.

Reply:
Previously ovn-chassis was put in it's own group, which was confusing and made the code unecessarily complex.

Reply:
The reason why we need this is because ovn-chassis needs to upgrade before ovn-central.

Reply:
nova-compute (data-plane) and octavia (control-plane) has ovn-chassis. I'm not seeing a good reason to not have subordinates as a group once they are already mixed

Reply:
Thanks for explanation.